cmd/read.go

```diff
@@ -52,10 +54,40 @@ If --pcrs is not provided, all pcrs are read for that hash algorithm.`,
 	},
 }
 
+var nvReadCmd = &cobra.Command{
+	Use:   "nvdata <endorsement | owner | platform | null>",
```

Comment:
For now, let's not bother with heierarchies for this command. We should just use the owner heierarchy and be done with it.

The only other heierarchy that can be used here is  the platfrom heierachy, and it generally wouldn't work with the empty password anyway. (as platform NV Indicies generally used enhanced auth).

Reply:
done